source/includes/steps-install-verify-files-windows.yaml

```diff
@@ -0,0 +1,129 @@
+title: Download the Signature Check utility from Microsoft.
+stepnum: 1
+level: 4
+ref: download-sigcheck
+content: |
+  a. Visit the `Sigcheck utility page <https://docs.microsoft.com/en-us/sysinternals/downloads/sigcheck>`__.
+
+  b. Click the :guilabel:`Download Sigcheck` link.
+
+  c. Unzip ``Sigcheck.zip``.
+
+  d. Move the Sigcheck directory to an appropriate location on your 
+     Windows host.
+
+     For this tutorial, this location is 
+     ``$Env:ProgramFiles\Sigcheck``.
```

Comment:
Is this a common notation? (I've never seen this before)

Reply:
Also isn't it usually "Program Files" or "Program Files (x86)"? Does this run the risk of confusing readers?

Reply:
This borrows from the [Microsoft Powershell Environment Variables](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-6) page and the [SS64 System Variables](https://ss64.com/nt/syntax-variables.html) page. As I was using Powershell, I wanted to use the variables just in case someone decided to configure their environment differently.

Reply:
Thanks for the info 👍